### DISCUSSION TITLE
Bindings to Golang and pre-processed planet tiles

### DISCUSSION THREAD

**Discussion by ghost** [valhalla/valhalla#4414] [Show and tell]
I made a [golang bindings](https://github.com/pufferffish/valhalla-go) a while back, valhalla is directly embedded into the library via cgo. I don't remember exactly but I think I encountered some issues with big HTTP requests being not-performant or straight up not working due to the request being too big, so rather than making a HTTP wrapper around valhalla, the library just directly calls valhalla via C FFI.

Also I pre-processed tiles data for the entire planet and uploaded the results to internet archive: https://archive.org/details/valhalla-planet-221219

**Comment by nilsnolde** [comment]
Feel free to PR to the public README and add your project to the bottom: https://github.com/valhalla/valhalla#related-projects

**Comment by khalidDaoud** [comment]
Hi @pufferffish 
any hope you can update the binding
